config_example.json

```diff
@@ -1165,6 +1165,49 @@
     }
    ]
   },
+  {
+    "name": "Oex",
+    "enabled": true,
+    "verbose": false,
+    "websocket": false,
+    "useSandbox": false,
+    "restPollingDelay": 10,
+    "httpTimeout": 15000000000,
+    "httpUserAgent": "",
+    "httpDebugging": false,
+    "authenticatedApiSupport": false,
+    "authenticatedWebsocketApiSupport": false,
+    "apiKey": "Key",
+    "apiSecret": "Secret",
+    "apiUrl": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+    "apiUrlSecondary": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+    "proxyAddress": "",
+    "websocketUrl": "NON_DEFAULT_HTTP_LINK_TO_WEBSOCKET_EXCHANGE_API",
+    "availablePairs": "btcusd",
```

Comment:
can you just load this in to your config.json file to test if this works. There are some pairs missing. and I think configCurrenyPairFormat might be configured incorrectly.

Reply:
Please update the list of available pairs, can see that a bunch are missing from here: https://openapi.oex.com/open/api/common/symbols